dotcom-rendering/src/web/components/Callout/CalloutComponents.tsx

```diff
@@ -1,47 +1,55 @@
 import { css } from '@emotion/react';
 import {
 	body,
+	brand,
 	neutral,
 	palette,
 	space,
+	success,
 	textSans,
 } from '@guardian/source-foundations';
-import { Button, SvgShareCallout } from '@guardian/source-react-components';
+import {
+	Button,
+	SvgShareCallout,
+	SvgTickRound,
+} from '@guardian/source-react-components';
 import { useState } from 'react';
-import { decidePalette } from '../../lib/decidePalette';
-
-const descriptionStyles = (format: ArticleFormat) =>
-	css`
-		a {
-			color: ${decidePalette(format).text.richLink};
-			border-bottom: 1px solid ${decidePalette(format).text.richLink};
-			text-decoration: none;
-		}
-		padding-bottom: ${space[4]}px;
-		${body.medium()}
 
-		p {
-			margin-bottom: ${space[3]}px;
-		}
-	`;
+const descriptionStyles = css`
+	a {
+		color: ${brand[500]};
+		border-bottom: 1px solid ${brand[500]};
+		text-decoration: none;
+	}
+	padding-bottom: ${space[4]}px;
+	${body.medium()}
+
+	p {
+		margin-bottom: ${space[3]}px;
+	}
+`;
 
 export const CalloutDescription = ({
 	description,
-	format,
 }: {
 	description: string;
-	format: ArticleFormat;
 }) => (
-	<div css={descriptionStyles(format)}>
+	<div css={descriptionStyles}>
 		<div dangerouslySetInnerHTML={{ __html: description }}></div>
 		<div>
 			Please share your story if you are 18 or over, anonymously if you
 			wish. For more information please see our{' '}
-			<a href="https://www.theguardian.com/help/terms-of-service">
+			<a
+				data-ignore="global-link-styling"
```

Comment:
This line here `data-ignore="global-link-styling"` is important as it ensures that the `a` tag styling that I wrote above is applied. Unfortunately, I don't know how to do this on line 38 where we use `dangerouslySetInnerHTML`